circle.yml

```diff
@@ -0,0 +1,66 @@
+version: 2
+
+jobs:
+  macos:
+    macos:
+      xcode: "10.1.0"
+    steps:
+      - checkout
+      - run: swift build
+      - run: swift test
+
+  linux:
+    docker:
+      - image: swift:4.2
+    steps:
+      - checkout
+      - run: 
+          name: Compile code
+          command: swift build
```

Comment:
we're probably ok to remove the `name:, command:` labels here and match the `macOS` section. Would be easier to read I think.